src/PalindromeNumber/Solution.cpp

```diff
@@ -0,0 +1,78 @@
+// Solution by Imtiaz Ahmed (Github: tiazahmd)
+
+#include <iostream>
+#include <vector>
+
+#include "../Test.h"
+
+using namespace std;
+using namespace leetcode;
+
+class Solution {
+public:
+  string isPalindrome(int x) {
+    if (x >= 0 && x < 10) {
+      return "true";
+    } else if (x >= 10) {
+      vector<int> vec;
+      long div = 10;
+      long ct = 1;
+      int num = 0;
+      int digCount = 0;
+      int temp = x;
```

Comment:
I think the `ct` and `digCount` are not necessary.
you can use a `while` loop to fill vector like this
```cpp
while (temp != 0) {
  num = temp % div;
  temp = temp / div;
  vec.push_back(num);
}
```

Reply:
Will make the changes.